scripts/deploy-diamond.ts

```diff
@@ -165,18 +173,40 @@ export const deployDiamond = async () => {
       action: FacetCutAction.Add,
       functionSelectors: getSelectors(ownershipFacet),
     },
-    ...appCuts,
   ];
 
-  // Encode the data for the init function call
-  // The DiamondInit.init() function will be called with delegatecall after the
-  // diamond cut is performed
-  const functionCall = diamondInit.interface.encodeFunctionData("init");
+  // Encode the data for the diamondInit init function call The
+  // DiamondInit.init() function will be called with delegatecall after the
+  // diamond cut for the diamond facets is performed
+  const diamondInitCallData = diamondInit.interface.encodeFunctionData("init");
+
+  // This adds the sarco token to AppStorage.
+  //
+  // Encode the data for appStorageInit init function call The
+  // AppStorageInit.init() function will be called with delegatecall after the
+  // diamond cut for the sarcophagus facets is performed.
+  const appInitCallData = appStorageInit.interface.encodeFunctionData("init", [
+    mockSarcoToken.address,
+  ]);
```

Comment:
When deploying to testnet / mainnet, would we be running this same script, but passing in the testnet / mainnet sarco token address? Or would a different script be run?

Reply:
This script should be used for deploying to testnet/mainnet. If not it would be very similar.